scripts/functional-tests.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 # SPDX-License-Identifier: Apache-2.0
+#
+# If you are running locally and calling the script multiple times you may want to run like this:
+#
+# CLEANUP_TEST_DIR=1 TEST_DIR=/tmp/foo ./scripts/functional-tests.sh
```

Comment:
The script is being triggered through `tox` environment, which I believe filters out variables from external envvars, unless they are listed in the corresponding `tox.ini` section as `passenv`. You may want to update `tox.ini` so that `tox -e py311-functional` honors the change.

Reply:
That's not how I run the func test locally, I call the script directly, am I doing something wrong? :)